Meadowlark-js/packages/meadowlark-core/src/handler/FrontendFacade.ts

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+import R from 'ramda';
+import { writeErrorToLog } from '../Logger';
+import { authenticate } from '../middleware/AuthenticationMiddleware';
+import { MiddlewareChain } from '../middleware/MiddlewareChain';
+import { parsePath } from '../middleware/ParsePathMiddleware';
+import { parseBody } from '../middleware/ParseBodyMiddleware';
+import { validateResource } from '../middleware/ValidateResourceMiddleware';
+import { noBody } from '../middleware/NoBodyMiddleware';
+import { FrontendRequest } from './FrontendRequest';
+import { FrontendResponse } from './FrontendResponse';
+import * as Upsert from './Upsert';
+import * as Update from './Update';
+import * as Delete from './Delete';
+import * as Get from './Get';
+
+// Middleware stack for methods with a body in the request (POST, PUT)
+const methodWithBodyStack: (chain: MiddlewareChain) => Promise<MiddlewareChain> = R.pipe(
```

Comment:
Two different middleware stacks, one for requests with body, one for requests without.